solr/core/src/test/org/apache/solr/cli/TestSolrCLIRunExample.java

```diff
@@ -70,6 +72,7 @@ public static void beforeClass() {
         Paths.get(".").toAbsolutePath().toString().contains(" "));
     // to be true
     System.setProperty("solr.directoryFactory", "solr.NRTCachingDirectoryFactory");
+    System.setProperty(LIB_ENABLED_SYSPROP, "true");
```

Comment:
If we remove the lib directives in the example ConfigSet, these should be able to be removed, right?

Reply:
Good catch, fixed!